site/src/pages/GroupsPage/SettingsGroupPageView.tsx

```diff
@@ -53,9 +65,60 @@ const UpdateGroupForm: React.FC<{
           onChange={onChangeTrimmed(form)}
           autoFocus
           fullWidth
-          label="Avatar URL"
+          label="Icon"
           variant="outlined"
+          InputProps={{
+            endAdornment: hasIcon ? (
+              <InputAdornment position="end">
+                <img
+                  alt=""
+                  src={form.values.avatar_url}
+                  className={styles.adornment}
+                  // This prevent browser to display the ugly error icon if the
+                  // image path is wrong or user didn't finish typing the url
+                  onError={(e) => (e.currentTarget.style.display = "none")}
+                  onLoad={(e) => (e.currentTarget.style.display = "inline")}
+                />
+              </InputAdornment>
+            ) : undefined,
+          }}
         />
+
+        <Button
+          fullWidth
+          ref={emojiButtonRef}
+          variant="outlined"
+          size="small"
+          endIcon={<OpenDropdown />}
+          onClick={() => {
+            setIsEmojiPickerOpen((v) => !v)
+          }}
+        >
+          {"Select emoji"}
```

Comment:
Can we translate this by adding it to `i18n`? There's a guide [here](https://github.com/coder/coder/pull/3682). Let me know if you have questions!

Reply:
I'd rather abstract this component first, which I could do as part of this PR, but since it's only been used twice, I think we wait for now. Is that alright?

Reply:
Lol, I'm not sure what that has to do with abstraction, but it's not a blocker.

Reply:
That was actually a questionable comment, I'll just use the same language thing in both...

Reply:
Fixed this btw